CONTRIBUTING.md

```diff
@@ -146,34 +146,66 @@ The process described here has several goals:
-
-While the prerequisites above may be satisfied, the reviewer(s) may still ask you to do additional work or ask for changes before your PR is finally accepted. 
-Since the reviewer knows the code very well, they might have ideas how to include features more elegantly or see inconsistencies compared to the code base. 
+3. Describe in your PR what you have done, what this is fixing or introducing and why this isn't regressive.
+If there is an issue to your PR, please link it.
+If you have adhered to the style guide of the [commit messages](#git-commit-messages), you will have that content already.
+4. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing.
+Should a status check fail and you believe it is unrelated to your contribution, please write so in a comment in the PR.
+
+While the prerequisites above may be satisfied, the reviewer(s) may still ask you to do additional work or ask for changes before your PR is finally accepted.
+Since the reviewer knows the code very well, they might have ideas how to include features more elegantly or see inconsistencies compared to the code base.
+
+#### Reviews
+Once your pull request is being reviewed, there are some guidelines.
+1. Threads started during a review should only be marked as resolved by the started of the thread.
+2. Comments and discussions should be ended cleanly. This can be done by using emoticons on the comments,
+e. g. a "thumbs-up" to mark a comment as done.
+3. Once you have integrated changes that were requested, you can dismiss the review of the reviewer and re-request a review.
+
+An approval of the PR means that the PR can be merged as is, without any changes still necessary.
+If the owner of a PR is an Eclipse committer of the project, the owner can merge their own PR once approved.
+Otherwise, the PR needs to be merged by an Eclipse committer of the project.
+
+#### Squashing
+Depending the size or content of a pull request it might be sensible to separate your pull request into several commits.
+However, several commits are not always necessary. Only separate commits if their contents belong separately.
+If a commit alone does not build, then it probably belongs with additional other changes.
+When changes are required fixup your commits or squash new commits based on requested changes.
+[Here you can find a guide on fixups and autosquashing](https://fle.github.io/git-tip-keep-your-branch-clean-with-fixup-and-autosquash.html).
+Do not forget to amend your commit messages along with the changes.
 
 ## Style Guides
 
 ### Git Commit Messages
 
-We use the commit style of the [SW360 project](https://github.com/eclipse/sw360/wiki/Dev-Semantic-Commits).
+We use the commit style of the [SW360 project](https://github.com/eclipse/sw360/wiki/Dev-Semantic-Commits) for commit message headlines.
```

Comment:
Haven't we decided to not use this commit message style? I am very unsure.

Reply:
I believe we had decided against it, yes, as it often takes up too much space in the commit message title. Personally, I like to refer to Git's own [submitting patches](https://github.com/git/git/blob/master/Documentation/SubmittingPatches#L43), which is also [what ORT refers to](https://github.com/heremaps/oss-review-toolkit/blob/master/CONTRIBUTING.md#git-history), to describe the style I believe we agreed on.